### ISSUE TITLE
link.DroppedAttributeCount undefined (type "go.opentelemetry.io/otel/trace".Link has no field or method DroppedAttributeCount

### ISSUE THREAD

**Issue by AnnatarHe** [uptrace/uptrace-go#75]
Description:

Cannot running with Gin plugin

Error:
```bash
/Users/username/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.0.0-!r!c1/trace/span.go:438:7: link.DroppedAttributeCount undefined (type "go.opentelemetry.io/otel/trace".Link has no field or method DroppedAttributeCount)
```

code sample:
```go
package main

import (
	"context"
	"errors"
	"fmt"

	"go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin"
	"go.opentelemetry.io/otel"
	"go.opentelemetry.io/otel/attribute"

	"github.com/gin-gonic/gin"
	"github.com/uptrace/uptrace-go/uptrace"
)

func main() {
	ctx := context.Background()

	uptrace.ConfigureOpentelemetry(&uptrace.Config{
		// copy your project DSN here or use UPTRACE_DSN env var
		DSN: "",

		ServiceName:    "test-myservice",
		ServiceVersion: "0.0.1",
	})
	router := gin.Default()
	router.Use(otelgin.Middleware("service-name"))
	defer uptrace.Shutdown(ctx)

	tracer := otel.Tracer("app_or_package_name")
	ctx, span := tracer.Start(ctx, "main")
	span.End()
	fmt.Printf("trace: %s\n", uptrace.TraceURL(span))
}
```

go.mod:
```go
module github.com/AnnatarHe/testuptrace

go 1.16

require (
	github.com/gin-gonic/gin v1.7.3
	github.com/uptrace/uptrace-go v0.21.1
	go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin v0.22.0
	go.opentelemetry.io/otel v1.0.0-RC2
)

```

**Comment by vmihailenco** [comment]
Hi,

We need to update uptrace-go to the latest opentelemetry-go version (v1.0.0-RC2). I've already updated the master branch and will release a new version tomorrow.

Thanks for reporting this :+1:

**Comment by vmihailenco** [comment]
This should be fixed in v1.0.0-RC2. Please update your go.mod file

```
	github.com/uptrace/uptrace-go v1.0.0-RC2
```

or use our [gin example](https://github.com/uptrace/uptrace-go/tree/master/example/gin)

**Comment by AnnatarHe** [comment]
Awesome!
I tried the new version of uptrace. It works very well.
Thanks 👍
